### DISCUSSION TITLE
Ampliar o escopo do projeto e fazer um sistema completo de gestão de personagens

### DISCUSSION THREAD

**Discussion by NightSpirit2099** [YuriAlessandro/gerador-ficha-tormenta20#332] [Ideias]
Seria muito bom fazer um sistema completo de gestão de personagens, ou pelo menos que possibilitasse criar e evoluir o personagem adicionando poderes etc.

**Comment by YuriAlessandro** [comment]
Isso era uma coisa que a gente estava evitando, porque o Orc&Roll já faz isso e ele é oficialmente suportado pelo pessoal da Jambô (que não é nosso caso). Mas tendo em vista que eles não atualizam a plataforma desde julho do ano passado e bastante gente tem pedido pra gente, nós vamos começar a fazer isso. Mas não deve ser algo tão rápido de finalizar.

**Reply by NightSpirit2099** [reply]
Obrigado pela resposta!

**Comment by YuriAlessandro** [comment]
Só para atualizar: eu comecei a desenvolver isso mas é BEM COMPLICADO mesmo. Eu dei uma pausa agora para focar nisso aqui: https://github.com/YuriAlessandro/gerador-ficha-tormenta20/discussions/330

**Comment by YuriAlessandro** [comment]
:) isso agora existe
